common/app/views/fragments/commercial/badge.scala.html

```diff
@@ -1,8 +1,24 @@
 @(content: model.ContentType)(implicit request: RequestHeader)
 @import common.Edition
-@import conf.switches.Switches.SponsoredSwitch
+@import conf.switches.Switches.{SponsoredSwitch, staticBadgesSwitch}
 @import model.{Article, Audio, Gallery, Interactive, Video}
 
+@if(staticBadgesSwitch.isSwitchedOn) {
+
+    @for(branding <- content.branding(Edition(request))) {
+        <div class="ad-slot--paid-for-badge ad-slot--paid-for-badge--article ad-slot--force-display">
```

Comment:
Let's avoid class names like these as they will be caught by opportunistic adblockers. Actually the whole HTML snippet could be much simplified using @uplne 's magic